lib/structuraid_core/design_codes/aci_318_19/rc/footings/bending_reinforcement_ratio.rb

```diff
@@ -0,0 +1,65 @@
+module StructuraidCore
+  module DesignCodes
+    module ACI31819
+      module RC
+        module Footings
+          class BendingReinforcementRatio
+            MINIMUM_RATIO = 0.0025
+            CODE_REFERENCE = 'ACI 318-19'.freeze
+
+            include DesignCodes::Utils::CodeRequirement
+            use_schema DesignCodes::Schemas::RC::Footings::BendingReinforcementRatioSchema
+
+            def call
+              [
+                solve_cuadratic_equation_for_steel_reinforcement_ratio,
+                MINIMUM_RATIO
+              ].max
+            end
+
+            private
+
+            ## the target is to solve [(1 - sqrt(1 - 4ac))/2a, (1 + sqrt(1 - 4ac))/2a]
+
+            def solve_cuadratic_equation_for_steel_reinforcement_ratio
+              [
+                equation_solver_option_1.negative? ? Float::INFINITY : equation_solver_option_1,
+                equation_solver_option_2
+              ].min
+            end
+
+            def equation_solver_option_1
+              (1 - equation_component_root) / (2 * equation_component_a)
+            end
+
+            def equation_solver_option_2
+              (1 + equation_component_root) / (2 * equation_component_a)
+            end
+
+            def equation_component_a
+              0.59 * design_steel_yield_strength / design_compression_strength
+            end
+
+            def equation_component_c
+              section_area = width * effective_height # m**2
+              reduced_steel_strength = capacity_reduction_factor * design_steel_yield_strength # N/(m**2)
+              flexural_moment / (reduced_steel_strength * section_area * effective_height) # N/(m**2)
```

Comment:
I think the unit comments are wrong
```suggestion
              section_area = width * effective_height # mm**2
              reduced_steel_strength = capacity_reduction_factor * design_steel_yield_strength # MPa
              flexural_moment / (reduced_steel_strength * section_area * effective_height) # MPa
```

Reply:
Let's wrap this comment into a fix due to an error. I was thinking m, Pa, and N